src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseDetailsEndpoint.java

```diff
@@ -308,6 +318,7 @@ public CaseDetails saveCaseDetailsForCitizen(
         @ApiResponse(code = 422, message = "Field validation failed"),
         @ApiResponse(code = 409, message = "Case reference not unique")
     })
+    @LogAudit(operationType = OperationType.CREATE_CASE)
     public JsonNode validateCaseDetails(
```

Comment:
we agreed not to log this

Reply:
We have initially committed this on v1 as well, and then asked questions. Will go to another PR.